rc

```diff
@@ -13,3 +13,16 @@ clwdiff() {
 # Useful for chaining cl commands.
 # Example: cl search .orphan -u | xcl info
 alias xcl='xargs cl'
+
+# Activate autocomplete
+# If you are using zsh, you will also need to include the following in your .zshrc:
+#   autoload -U +X compinit && compinit
+#   autoload -U +X bashcompinit && bashcompinit
+
+if [ -z ${CODALAB_PATH+x} ]; then
```

Comment:
Let's use `dirname $0` to figure out the CODALAB_PATH so we don't have to have users set it manually.